### ISSUE TITLE
@Transactional annotation not support attach other annotations

### ISSUE THREAD

**Issue by icode** [ebean-orm/avaje-ebeanorm-agent#14]
```
@Path("a")
public class LocalResource {

    @Transactional
    @Path("b")
    public String doSomething() {
        return "LocalResource do something ...";
    }
}
```
than LocalResource.doSomething method @Path not have value

**Comment by icode** [comment]
please fix it ! @rbygrave

**Comment by icode** [comment]
@rbygrave I need Transactional please check and fix it!

**Comment by rbygrave** [comment]
I should be able to try and reproduce this tonight.

Cheers, Rob.

**Comment by rbygrave** [comment]
Fixed in 4.5.3
